test/integration/test_kubernetes_staging.py

```diff
@@ -73,35 +105,65 @@ def job_config(jobs_directory):
 
 @integration_util.skip_unless_kubernetes()
 @integration_util.skip_unless_fixed_port()
-class BaseKubernetesStagingIntegrationTestCase(BaseJobEnvironmentIntegrationTestCase, MulledJobTestCases):
+class KubernetesStagingContainerIntegrationTestCase(BaseJobEnvironmentIntegrationTestCase, MulledJobTestCases):
 
     def setUp(self):
-        super(BaseKubernetesStagingIntegrationTestCase, self).setUp()
+        super(KubernetesStagingContainerIntegrationTestCase, self).setUp()
         self.history_id = self.dataset_populator.new_history()
 
     @classmethod
     def setUpClass(cls):
         # realpath for docker deployed in a VM on Mac, also done in driver_util.
         cls.jobs_directory = os.path.realpath(tempfile.mkdtemp())
-        super(BaseKubernetesStagingIntegrationTestCase, cls).setUpClass()
+        super(KubernetesStagingContainerIntegrationTestCase, cls).setUpClass()
 
     @classmethod
     def handle_galaxy_config_kwds(cls, config):
         config["jobs_directory"] = cls.jobs_directory
         config["file_path"] = cls.jobs_directory
-        config["job_config_file"] = job_config(cls.jobs_directory)
-
+        config["job_config_file"] = job_config(CONTAINERIZED_TEMPLATE, cls.jobs_directory)
         config["default_job_shell"] = '/bin/sh'
-        # Disable tool dependency resolution.
+        # Disable local tool dependency resolution.
         config["tool_dependency_dir"] = "none"
-        config["enable_beta_mulled_containers"] = "true"
```

Comment:
Ah, that's why the coexecution test fails, this is still needed.

Reply:
I'm not wrong... Galaxy is wrong https://github.com/galaxyproject/galaxy/pull/8569 😆. I don't know why I dropped this probably was testing some specific test and wanted to isolate the problem. I have been meaning to do #8569 for some time though.

Reply:
Haha, OK, can you rebase this against dev or #8569? I'll merge this then.

Reply:
Once the tests pass for #8569 I'll merge that and then rebase this.

Reply:
Merge now? Rightly Martin doesn't like it when I merge a PR I've force pushed since the last approve 😅.